src/test/resources/hw6/exercise1/exercise1.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Exercise 1
+
+  Scenario: Check different pages
+    Given I open EPAM JDI Site
+    Then browser title should be 'Home Page'
+    When I login as 'epam' with password '1234'
+    Then username should be 'PITER CHAILOVSKII' on the Home page
+    And 4 pictures are displayed on the Home Page
+    And 4 text are displayed under pictures on the Home Page
+    And headline above the pictures is displayed on the Home Page
+    And description above the pictures is displayed on the Home Page
+    When I click on 'Service' in the header menu
```

Comment:
On which page? If you use page name it is better to use them in all steps

Reply:
fix